dbms/src/Interpreters/Aggregator.cpp

```diff
@@ -1348,27 +1340,23 @@ void NO_INLINE Aggregator::mergeDataOnlyExistingKeysImpl(
     if constexpr (Method::low_cardinality_optimization)
         mergeDataNullKey<Method, Table>(table_dst, table_src, arena);
 
-    for (auto it = table_src.begin(); it != table_src.end(); ++it)
+    table_src.mergeToViaFind(table_dst,
+        [&](AggregateDataPtr dst, AggregateDataPtr & src, bool found)
```

Comment:
indent

Reply:
How should we change the indentation here?

Reply:
it's the brace after this line, the block should have the same indentation to lambda

Reply:
or you can pull the lambda up like this https://github.com/yandex/ClickHouse/blob/master/dbms/programs/server/Server.cpp#L452

Reply:
Moved it to the same line.